src/Render/Program.cs

```diff
@@ -88,11 +88,18 @@ internal static class Program {
 		}
 
 		private static string GetTitle( MarkdownDocument doc ) {
-			var inline = doc.Descendants<HeadingBlock>().Single(
-				h => h.Level == 1
-			).Inline;
-			var titleLiteral = inline.Descendants<LiteralInline>().Single();
-			return titleLiteral.Content.ToString();
+			try {
+				var inline = doc.Descendants<HeadingBlock>().Single(

```

Comment:
Consider using [`SingleOrDefault`](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.singleordefault?view=netcore-3.1) instead of catching the exception. Mostly personal preference, but I opt for [preventing an exception instead of catching it after it's thrown](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#handle-common-conditions-without-throwing-exceptions), unless the exception contains information that's necessary to expose (for logging, etc.).